integration_tests/src/main/python/asserts.py

```diff
@@ -90,7 +90,12 @@ def _assert_equal(cpu, gpu, float_check, path):
 
 def assert_equal(cpu, gpu):
     """Verify that the result from the CPU and the GPU are equal"""
-    _assert_equal(cpu, gpu, float_check=get_float_check(), path=[])
+    try:
+      _assert_equal(cpu, gpu, float_check=get_float_check(), path=[])
+    except:
+      print("CPU OUTPUT: %s" % cpu)
+      print("GPU OUTPUT: %s" % gpu)
```

Comment:
nit: technically this is not guaranteed to be the output for the GPU or the CPU because we could have sorted it locally first, but I think everyone will understand and I think this is the right place to print out the info.